### PR TITLE
chore: document how to install dependencies

### PR DESCRIPTION
**Pull request by madmikeross**
This change adds a brief explanation to the README.md to document how to install the project's node dependencies.

### CODE REVIEW

**Comment by JeffwPotter** [comment]
pretty shit tbh

File: README.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
 # lambda-budget
 Simple budgeting app
+
+## Installing dependencies
+After you have cloned the repo, install node dependencies by running the following command:
```
Comment (JTBach) [inline README.md:5]: You dont know how to spell

File: README.md [hunk 2]
```diff
@@ -1,2 +1,8 @@
 # lambda-budget
 Simple budgeting app
+
+## Installing dependencies
```
Comment (madmikeross) [inline README.md:4]:
```suggestion
## Installing Node dependencies
```

Comment (JTBach) [reply README.md:4]: Great change

**Review by JeffwPotter — APPROVED** [review]

**Review by JTBach — APPROVED** [review]
Grood Stuff

**Review by JeffwPotter — APPROVED** [review]

**Pull request merged**